internetarchive/openlibrary
# remove unused around_the_library.html and get_random_recent_changes

Closes #9896


refactor and cleanup

### Technical

- deleted the unused file `around_the_library.html` and the function `get_random_recent_changes` which is present in the `openlibrary/plugins/upstream/utils.py`
- Verified that `around_the_library.html` and `get_random_recent_changes` were not referenced or called anywhere else in the codebase


### Testing

1. Verify that the file `openlibrary/templates/site/around_the_library.html` has been deleted
2. Check `openlibrary/plugins/upstream/utils.py` to confirm that the function `get_random_recent_changes` has been removed
3. Run the application and ensure no errors occur related to these deletions


### Stakeholders

@scottbarnes
